frontend/tests/utils/helpers.ts

```diff
@@ -0,0 +1,11 @@
+import type { Page as playwrightPage } from 'playwright-core'
+import {
+  expect as baseExpect, test as baseTest,
```

Comment:
better to be specific here
```suggestion
  expect as playwrightExpect, test as playwrightTest,
```